lib/aptible/cli/helpers/maintenance.rb

```diff
@@ -0,0 +1,20 @@
+require 'aptible/api'
+
+module Aptible
+  module CLI
+    module Helpers
+      module Maintenance
+        include Helpers::Token
+
+        def maintenance_apps
+          Aptible::Api::MaintenanceApp.all(token: fetch_token)
```

Comment:
1. can filter this (instead of in the view layer) if maintenance_deadline available
2. can hydrate it with additional app / db information with a new possible object (or paired object), so we can group by account // include other info (unsure if desired)